### PR TITLE
run on staging branch

### PR DESCRIPTION
**Pull request by morenol**

### CODE REVIEW

**Review by simlay — APPROVED** [review]

File: src/admin.rs [hunk 1]
```diff
@@ -150,7 +150,7 @@ impl FluvioAdminJS {
 
     #[node_bindgen]
     async fn list_spu(&mut self) -> Result<ArrayBuffer, FluvioErrorJS> {
-        serde::__private::Ok(self.js_list::<SpuSpec>().await?)
```
Comment (simlay) [inline src/admin.rs:153]: wat? I didn't even know this was in here.

**Review by simlay — APPROVED** [review]

File: src/admin.rs (hunk 1 above)
Comment (morenol) [reply src/admin.rs:153]: I have seen this in the past suggested by code editor autocompletion

**Comment by morenol** [comment]
`check formatting (ubuntu-latest, stable, 14)` and `Smoke test (ubuntu-latest, stable, 12)` are marked as required, @simlay do you have permissions in the repo setting to remove them? It looks like it should be `CI / Smoke test (ubuntu-latest, stable, 16` and not sure babout the formatting, maybe `CI / Check fmt and clippy`

File: src/admin.rs (hunk 1 above)
Comment (simlay) [reply src/admin.rs:153]: I've seen it in come build errors in very rare scenarios. I've fixed them with a `cargo clean` and a `cargo build`.

**Pull request merged**